denylist/src/client.rs

```diff
@@ -0,0 +1,47 @@
+use crate::{error::Error, models::metadata::DenyListMetaData, Result};
+use std::{str, time::Duration};
+
+/// The default client useragent for denylist http requests
+static USERAGENT: &str = "oracle/poc_iot_verifier/1.0";
+/// The default timeout for http requests
+pub const DEFAULT_TIMEOUT: Duration = Duration::from_secs(5);
+
+#[derive(Clone, Debug)]
+pub struct DenyListClient {
+    pub client: reqwest::Client,
+}
+
+// TODO: basic non production client. productionise....
```

Comment:
what needs to be done to productionise this?

Reply:
..oh just have to remove the TODO. Its all good now.